### ISSUE TITLE
Legacy Users in Germany Log on bug

### ISSUE THREAD

**Issue by Gargoyle112** [openaudible/openaudible#591]
Info:
-Legacy User
-Audible.de

When I try to connect to Audible, I select the german database and select "legacy user".
A new Windows pops up and I enter my credentials (username and password) and click continue.
After that I have to reenter my credentials and a captcha.

As soon as I click on continue, I get a popup from Openaudible:

Title:
Error Registering Connection

Content:
[Red Error Cross] Unable to register a device.

After I click OK, OpenAudible asks me if I want to sync my library.
When I cklick on Yes, I get this error:

Title:
Error

Content:
[Red Error Cross] Error refreshing library. Not logged in.

**Comment by openaudible** [comment]
Legacy accounts from Germany may be broken. I assume you have an old user name that is not your Amazon email. 

Try restart app, log in, after error, send a bug report with this issue number. Maybe we can spot the problem.

**Comment by Gargoyle112** [comment]
Bug report sent

**Comment by DrRipCurl** [comment]
Hello,
I have exactly the same problem as "Gargoyle112" .
legacy user - Germany - Audible.de
same problem withe connecting to audible
[Red Error Cross] Unable to register a device.

and it is no problem to login on the audible website with my account, so....

what can I do?

**Comment by Gargoyle112** [comment]
Another thing  I found out:

The Error doesn't occur with version 2.3.5

**Comment by mkb79** [comment]
@Gargoyle112 @DrRipCurl 

Can you try my Audible app for Python? When you have installed it please execute these Python code:

```python
import audible

username = 'FILLMEOUT'
password = 'FILLMEOUT'
auth_file = 'TARGETOUTFILE.json'


auth = audible.Authenticator.from_login(
    username=username,
    password=password,
    locale='de',
    with_username=True)
auth.register_device()
# if you want to reuse the session
auth.to_file(auth_file)
# otherwise deregister now
auth.deregister_device()
```

**Comment by openaudible** [comment]
@Gargoyle112 Thanks for your help! 2.3.5 uses a web scraper to access your library. 3.0 uses the new login for faster and more accurate access (but has the problem with legacy users in .DE)

**Comment by mkb79** [comment]
I‘ve found the solution for this issue. Will report this to @openaudible via PN

**Comment by openaudible** [comment]
Thank you @mkb79 !! 
A new beta has been posted to https://openaudible.org/beta

Give it a try and let us know..

**Comment by Gargoyle112** [comment]
Unfortunately still the same error.
Error report sent.

**Comment by openaudible** [comment]
Shoot. I saw another problem. Just posted a new version.. 3.0.1. Thank you for the help!

**Comment by Gargoyle112** [comment]
still the error. Report is already on the way.

Am Di., 23. März 2021 um 01:19 Uhr schrieb openaudible <
***@***.***>:

**Comment by openaudible** [comment]
OK! Fixed... Thanks @mkb79 and @Gargoyle112 .  
https://openaudible.org/beta

Will be fixed in 3.0.1 final whenever that is released.

**Comment by Gargoyle112** [comment]
Works perfectly for me!
